docs/modules/ROOT/pages/mcp.adoc

```diff
@@ -0,0 +1,33 @@
+= Model Context Protocol
```

Comment:
I think we should also link to https://docs.quarkiverse.io/quarkus-mcp-server/dev/index.html

Reply:
Good point, I've added a note about it